### PR TITLE
fix/grants-1896

### PR DESCRIPTION
**Pull request by yunusfirat**

### CODE REVIEW

File: app/routes/start.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const copyprovider = (key) => {
     18: "Problems using the online service",
     19: "If you have any problems using the online service, contact the RPA.",
     20: "Telephone",
-    21: "Telephone: 0300 0200 301",
+    21: "Telephone: 03000 200 301",
```
Comment (zavies97) [inline app/routes/start.js:28]: Does this need to be updated?

Comment (yunusfirat) [reply app/routes/start.js:28]: Well, they did not raise bug for this but I received an email and realised that liam has been updated that part. 
here is the link https://eaflood.atlassian.net/browse/GRANTS-1816

Comment (yunusfirat) [reply app/routes/start.js:28]: Well, they did not raise bug for this but I received an email and realised that liam has been updated that part. 
here is the link https://eaflood.atlassian.net/browse/GRANTS-1816

Comment (yunusfirat) [reply app/routes/start.js]: I have just spoken to Liam now he said he did it accidentally so I will revert my changs.

**Review by diya2214 — APPROVED** [review]

**Pull request merged**